docs/chart_tests.md

```diff
@@ -34,19 +34,33 @@ mariadb/
   templates/
   templates/mariadb-tests.yaml
```

Comment:
Fixed. Also removed the `LICENSE` reference as that file isn't the chart's folder.